routes/api/articles.js

```diff
@@ -394,6 +422,59 @@ router.get(
  */
 router.put('/reaction/:articleId/', jwtValidator, verifyArticleId, validateReaction, likeOrDislike);
 
+/**
+ * @swagger
+ *
+ * /:articleId/report:
+ *   post:
+ *     description: To report an article
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: report Tag
+ *         description: The purpose of the report
+ *         in:  body
+ *         required: true
+ *         type: string
+ *         format: string
+ *       - name: message
+ *         description: The content of the report
+ *         in:  body
+ *         required: true
+ *         type: string
+ *         format: TEXT
+ *     responses:
+ *        - 200:
+ *          description: Report
+ *          message: Report was successfully created. Thank you.
+ *          Data: Null
+ *        - 500:
+ *          description: Server Error
+ *          message: There as been a server error
+ */
+router.post('/:articleId/report', jwtValidator, ReportController.createAReport);
+
+/**
+ * @swagger
+ *
+ * /admin/reports:
+ *   get:
+ *     description: To get all reports
+ *     produces:
+ *       - application/json
+ *     responses:
+ *        - 200:
+ *          description: report
+ *          message: Report retrieved successfully.
+ *          Data: reports
+ *        - 500:
+ *          description: Server Error
+ *          message: There as been a server error
+ *
+ *
+ */
+router.get('/admin/reports', jwtValidator, ReportController.getReports);
```

Comment:
You don't have to add `admin` to the route URL because it is only accessible by an admin. What happens in the future when other roles can access this endpoint?

Reply:
Fixed